### PR TITLE
mv.py

### PR DESCRIPTION
**Pull request by brianyu28**
Matcher for `mv` overwrite

### CODE REVIEW

**Comment by brianyu28** [comment]
Looks like we get output like:

```
$ mv bar baz/bar
mv: overwrite ‘baz/bar’?
```
which could allow us to infer that such a command is a movement rather than a renaming. However, a situation like:

```
$ mv baz/foo baz/bar
mv: overwrite ‘baz/bar’?
```

would produce the same message from `mv`, but it's a renaming. Any good way to handle that?

**Comment by dmalan** [comment]
Hm, how about "renaming" by default, else "moving" if the destination has a slash, even though the latter might still be a renaming? Guessing the example you proposed won't actually happen often in practice.

**Comment by brianyu28** [comment]
Added the logic for the distinction in the most recent commit.

File: helpers/mv.py [hunk 1]
```diff
@@ -3,11 +3,22 @@ def help(lines):
...
-            "Typing `n` and pressing return will cancel the operation."
-        ]
+        
+        # if a "/" is present in the destination path, assume moving a file; otherwise, renaming a file
+        moving = "/" in matches.group(1)
+        
+        interpretation = "You are "
+        interpretation += "moving" if moving else "renaming"
+        interpretation += " a file to `{}`, but there is already a file with the same name".format(matches.group(1))
+        interpretation += " there." if moving else " in the current directory."
```
Comment (dmalan) [inline helpers/mv.py]: Minor but seems inconsistent to have a trailing white space in your first string but then leading white space later?

**Comment by brianyu28** [comment]
Fixed.

**Pull request merged**